src/components/SideBar/PersonalList.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div class="personal-list">
+    <div
+      class="personal-list-detail"
+      v-for="list in lists"
+      :key="list.id"
+      :class="{
+        'selected-list': $store.selectedListId === list.id,
+      }"
+    >
+      <div class="personal-list-detail-left">
+        <i class="personal-list-detail-icon material-icons">
+          list
+        </i>
+        <input
+          v-if="$store.editting.listId === list.id"
+          :value="list.name"
+          type="text"
+          v-focus
+          @keyup.enter="$event.target.blur()"
+          @blur="editListName(list.id, $event)"
+        />
+        <p class="list-name" v-else @click="$store.selectedListId = list.id">
+          {{ list.name }}
+        </p>
+      </div>
+      <div class="personal-list-detail-right">
+        <p>{{ $store.countAllTasksBylistId(list.id) }}</p>
+        <i class="kebab-menu-icon material-icons">
+          more_vert
+        </i>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import Vue from "vue";
+export default {
+  name: "PersonalLists",
+  directives: {
+    focus: {
+      inserted(el) {
+        Vue.nextTick(() => el.focus());
+      },
+    },
+  },
+  props: {
+    lists: {
+      type: Array,
+    },
+  },
+  methods: {
+    editListName(listId, $event) {
+      const listName = $event.target.value.toString().trim();
+      if (!listName) {
+        return alert("List name required!");
+      }
+
+      const duplicated = !!this.lists.find(
+        (list) => list.id !== listId && list.name === listName
+      );
+
+      if (duplicated) {
+        return alert("Duplicated name!");
+      }
+
+      const isSuccess = this.$store.editListName(listId, listName);
+      if (isSuccess) return alert("There is something wrong!");
```

Comment:
`editListName()` doesn't return boolean in all possible cases, so this variable type should be checked.
And, such a magical logic when showing an error message after a successful action!

Reply:
Updated!